docs/src/content/docs/sdk.mdx

```diff
@@ -37,21 +75,73 @@ The session status is based on the `UltravoxSessionStatus` enum and can be one o
 | thinking      | The model is processing/thinking.           |
 | speaking      | The model is speaking.                      |
 
+#### Status Events
 The status can be retrieved by adding an event listener to the state:
 
-```javascript
-import { UltravoxSession } from 'ultravox-client';
+<Tabs syncKey="example-language">
+  <TabItem label="JavaScript">
+    ```js
+    import { UltravoxSession } from 'ultravox-client';
 
-const session = new UltravoxSession();
-const state = await session.joinCall('wss://your-call-join-url');
+    const session = new UltravoxSession();
+    const state = await session.joinCall('wss://your-call-join-url');
 
-// Listen for status changing events
-state.addEventListener('ultravoxSessionStatusChanged', (event) => {
-  console.log('Session status changed: ', event.state);
-});
+    // Listen for status changing events
+    state.addEventListener('ultravoxSessionStatusChanged', (event) => {
+      console.log('Session status changed: ', event.state);
+    });
 
-session.leaveCall();
-```
+    session.leaveCall();
+    ```
+  </TabItem>
+
+  <TabItem label="Flutter">
+    ```dart
+    import 'package:ultravox_client/ultravox_client.dart';
+
+    UltravoxSession session = UltravoxSession.create();
+    UltravoxSessionState state = await _session.joinCall(joinUrl);
+    state.addListener(() {
+      print('Status is now ${state.status}');
+    })
+    await session.leaveCall();
+
+    ```
+  </TabItem>
+
+  <TabItem label="Kotlin">
+    ```kotlin
+    import ai.ultravox.UltravoxSession
+
+    val session = UltravoxSession()
+    val sessionState = session.joinCall("wss://your-call-join-url")
+
+    sessionState.addEventHandler("ultravoxSessionStatusChanged") { event ->
```

Comment:
This is incorrect for Kotlin. There's no event parameter. You have to retrieve the values you want off of `state` as you do in Flutter and Python.  (In fact, I should probably change this in the JS SDK - I'll send you a PR for that later.)

Reply:
removing for now until i can review the other PR then will add back in